lib/backends/etcd.js

```diff
@@ -2,11 +2,10 @@
 'use strict'
 
 const etcdjs = require('etcdjs')
-const hash = require('../utils/hash')
 
 module.exports = class Etcd {
   constructor (path, opts) {
-    this.path = hash(path)
+    this.path = path.startsWith('.') ? path.slice(1) : path
```

Comment:
this is necessary because a leading dot character in etcd has meaning, it creates a "hidden" directory which is not a thing we need today nor a thing we should care to do. hidden keys in etcd are not a solution to anything.

Reply:
There is no better way to do this, because even if we change it to an underscore or something instead of a dot, to try to prevent something asking for `.foo` and `foo` colliding, all that would introduce is something asking for `.foo` and `_foo` colliding.

We are just going to have to either say "don't use dots" and accept them if they're there, or say "don't use dots" and convert them if they're there and then say "be aware of other paths that have stripped dots"

I think I talked myself out of recommending we just allow people to use dots in the code, but avoiding it in policy?